src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2716,6 +2716,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
         End Function
 
+        Public Overrides Function GetRequiredLanguageVersion(diagnostic As Diagnostic) As String
+            ' The driving scenario for this API (upgrading project based on language versions reported in diagnostics)
```

Comment:
Should this be throwing not supported? My usual mantra of "the compiler is an API so think about the API" seems to be getting violated here.